src/applications/appeals/995/tests/utils/helpers.unit.spec.js

```diff
@@ -24,21 +24,21 @@ import {
 } from '../../../shared/utils/issues';
 
 describe('getEligibleContestableIssues', () => {
-  const date = moment().startOf('day');
+  const date = startOfDay(new Date());
```

Comment:
I was exploring date-fns today and found a `startOfToday` function :D - no need to update this though